docs/src/pages/components/data-grid/getting-started/migration-v4.md

```diff
@@ -0,0 +1,434 @@
+  | `.MuiDataGrid-renderingZone`       | `.MuiDataGrid-virtualScrollerRenderZone` |
+  | `.MuiDataGrid-gridMenuList`        | `.MuiDataGrid-menuList`                  |
+  | `.MuiGridToolbarContainer-root`    | `.MuiDataGrid-toolbarContainer`          |
+  | `.MuiGridMenu-root`                | `.MuiDataGrid-menu`                      |
+  | `.MuiDataGridColumnsPanel-root`    | `.MuiDataGrid-columnsPanel`              |
+  | `.MuiGridPanel-root`               | `.MuiDataGrid-panel`                     |
+  | `.MuiGridPanelContent-root`        | `.MuiDataGrid-panelContent`              |
+  | `.MuiGridPanelFooter-root`         | `.MuiDataGrid-panelFooter`               |
+  | `.MuiDataGridPanelHeader-root`     | `.MuiDataGrid-panelHeader`               |
+  | `.MuiGridPanelWrapper-root`        | `.MuiDataGrid-panelWrapper`              |
+  | `.MuiGridFilterForm-root`          | `.MuiDataGrid-filterForm`                |
+  | `.MuiGridToolbarFilterButton-root` | `.MuiDataGrid-toolbarFilterList`         |
+
+### Module augmentation
+
+- The module augmentation is no longer enabled by default. This change was done to prevent conflicts with projects using `DataGrid` and `DataGridPro` together.
+
+  In order to still be able to do overrides at the theme level, add the following imports to your project:
+
+  ```diff
+  +import type {} from '@mui/x-data-grid/themeAugmentation';
+  +import type {} from '@mui/x-data-grid-pro/themeAugmentation';
+  ```
+
+### Virtualization
+
+- The `onViewportRowsChange` prop and the `viewportRowsChange` event have been removed
+
+  You can replace them with a listener on the `rowsScroll` event:
+
+  ```tsx
+  const apiRef = useGridApiRef();
+  const prevRenderContext = React.useRef(null);
+
+  React.useEffect(() => {
+    return apiRef.current.subscribeEvent('rowsScroll', ({ renderContext }) => {
+      if (
+        !prevRenderContext.current ||
+        renderContext.firstRowIdx !== prevRenderContext.current.firstRowIndex ||
+        renderContext.lastRowIdx !== prevRenderContext.current.lastRowIndex
+      ) {
+        prevRenderContext.current = renderContext;
+        const params = {
+          firstRowIndex: renderContext.firstRowIndex,
+          lastRowIndex: renderContext.lastRowIndex,
+        };
+      }
+    });
+  }, [apiRef]);
+
+  <DataGridPro apiRef={apiRef} />;
+  ```
+
+### Removed props
+
+- Some event listeners and DOM attributes were removed from `GridCell` and `GridRow` to improve performance.
+
+  The following props were removed.
+
+  - `onCellBlur`
+  - `onCellOver`
+  - `onCellOut`
+  - `onCellEnter`
+  - `onCellLeave`
+  - `onRowOver`
+  - `onRowOut`
+  - `onRowEnter`
+  - `onRowLeave`
+
+  If you depend on them, you can use `componentsProps.row` and `componentsProps.cell` to pass custom props to the row or cell.
+  For more information, check [this page](https://mui.com/components/data-grid/components/#row). Example:
```

Comment:
Please don't use absolute links, they get outdated once we version the docs (e.g. once put on https://v5.mui.com/).

```suggestion
  For more information, check [this page](/components/data-grid/components/#row). Example:
```

Reply:
I saw the same problem in https://github.com/mui-org/material-ui-x/pull/2676/files#diff-336bfe968fa69c66f1e3bed49cb09212756eec7e4355b782122920555c91ae17R17